components/article/stylesheets/index.styl

```diff
@@ -207,15 +207,18 @@ body.body-article
     margin auto
   figcaption
     margin-top 10px
-    color gray-dark-color
     text-align left
+    color gray-darker-color
+    font-size 15px
```

Comment:
When possible try to use the `garamond-size('l-caption')` this makes it easy for us to make global changes down the line. Not a blocker tho.